### DISCUSSION TITLE
[Bug] waf-detect template - no templates were found

### DISCUSSION THREAD

**Discussion by righettod** [projectdiscovery/nuclei-templates#1598] [Q&A] [answered]
**Nuclei version**

2.3.7

**Nuclei template version**

8.3.1

**Describe the bug**

The [waf-detect.yaml](https://github.com/projectdiscovery/nuclei-templates/blob/master/technologies/waf-detect.yaml) raised the following error:

**Commands to reproduce**

```
$ nuclei -t $NUCLEI_TPL_HOME/technologies/waf-detect.yaml -u https://righettod.eu
[INF] Loading templates...
[INF] Loading workflows...
[FTL] Error, no templates were found.
```

**Screenshots**

![image](https://user-images.githubusercontent.com/1573775/120289928-1cf20880-c2c2-11eb-8541-f6bae3b1de5f.png)

Does this message just indicate that no WAF was identified ?

Thanks a lot in advance 😃

**Comment by ehsandeep** [comment, marked as answer]
@righettod just add `-tags misc` in order to run this template, before it was running as default causing unwanted blocks by WAF, we updated and excluded this template from the default run.

**Reply by righettod** [reply]
Thanks a lot for your reply 😃
